### ISSUE TITLE
Gourmand crashes on launch after updating Python/Pip packages

### ISSUE THREAD

**Issue by rhymeswithmogul** [GourmandRecipeManager/gourmand#157]
I am no longer able to launch Gourmand after updating all of my Python and Pip packages.  I'm locked out of my recipe collection until I can figure out how to get this app working again.  Removing and reinstalling does not solve the problem.

### Steps to Reproduce

1. Open Gourmand.  (I'm adding the `-v` command-line switch for more information.)

### Expected Behavior
The app should launch and let me see my database.

### Current Behavior

It doesn't launch.  No GUI appears.  The app crashes with the following (verbose) output and backtrace: 

```
args = Namespace(db_url='', threads=False, gourmanddir='', thread_debug_interval=5.0, thread_debug=False, debug_file='', time=False, debug=1)
DEBUG_LEVEL= 1
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 87 Loading plugins from /home/myusername/.gourmand/plugins
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 87 Loading plugins from /usr/lib/python3/dist-packages/gourmand/plugins
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/listsaver.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/duplicate_finder.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/email.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/spellcheck.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/shopping_associations.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/nutritional_information.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/unit_converter.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/key_editor.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/unit_display_prefs.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/utf16.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/field_editor.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/browse_plugin.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 87 Loading plugins from /usr/lib/python3/dist-packages/gourmand/plugins/import_export
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/import_export/webimport.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/import_export/archive.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/import_export/mealmaster.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/import_export/plaintext.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/import_export/html.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/import_export/gxml.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/import_export/mastercook_plugin.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/import_export/mycookbook_plugin.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/import_export/pdf.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/import_export/epub.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/import_export/website_import.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 90 Found /usr/lib/python3/dist-packages/gourmand/plugins/import_export/krecipe_plugin.gourmet-plugin
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 101 Loading plugins from namespace
Plugin  copy_paste_plugin not found
DEBUG:   /usr/lib/python3/dist-packages/gourmand/backends/db.py: 185 Initializing DB connection
DEBUG:   /usr/lib/python3/dist-packages/gourmand/backends/db.py: 228 Done initializing DB connection
DEBUG:   /usr/lib/python3/dist-packages/gourmand/plugin_loader.py: 211 Instantiate <class 'unit_display_prefs.UnitDisplayDatabasePlugin'> from unit_display_prefs
Problem updating plugin <unit_display_prefs.UnitDisplayDatabasePlugin object at 0x7f8319291510>
Traceback (most recent call last):
  File "/usr/bin/gourmand", line 33, in <module>
    sys.exit(load_entry_point('gourmand==1.0.0', 'gui_scripts', 'gourmand')())
  File "/usr/lib/python3/dist-packages/gourmand/main.py", line 617, in launch_app
    RecGui.instance()
  File "/usr/lib/python3/dist-packages/gourmand/main.py", line 837, in instance
    RecGui.__single = cls()
  File "/usr/lib/python3/dist-packages/gourmand/main.py", line 846, in __init__
    GourmandApplication.__init__(self)
  File "/usr/lib/python3/dist-packages/gourmand/main.py", line 69, in __init__
    self.setup_recipes() # Setup recipe database
  File "/usr/lib/python3/dist-packages/gourmand/main.py", line 146, in setup_recipes
    self.rd = recipeManager.default_rec_manager()
  File "/usr/lib/python3/dist-packages/gourmand/recipeManager.py", line 84, in default_rec_manager
    return get_recipe_manager(**dbargs)
  File "/usr/lib/python3/dist-packages/gourmand/recipeManager.py", line 81, in get_recipe_manager
    return RecipeManager.instance_for(**kwargs)
  File "/usr/lib/python3/dist-packages/gourmand/backends/db.py", line 1861, in instance_for
    cls._instance_by_db_url[url] = cls(file, custom_url)
  File "/usr/lib/python3/dist-packages/gourmand/backends/db.py", line 1867, in __init__
    self.rd = get_database(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/gourmand/backends/db.py", line 2144, in get_database
    return RecData.instance_for(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/gourmand/backends/db.py", line 152, in instance_for
    cls._instance_by_db_url[url] = cls(file, url)
  File "/usr/lib/python3/dist-packages/gourmand/backends/db.py", line 175, in __init__
    self.update_version_info(gourmand.__version__.version)
  File "/usr/lib/python3/dist-packages/gourmand/backends/db.py", line 664, in update_version_info
    self.update_plugin_version(plugin,
  File "/usr/lib/python3/dist-packages/gourmand/backends/db.py", line 695, in update_plugin_version
    plugin.update_version(
TypeError: DatabasePlugin.update_version() got an unexpected keyword argument 'gourmand_stored'
```

### Environment

* Operating System: Ubuntu 22.04.3 LTS
* Version or commit id: 1.0.0
* Installed as Flatpak or Python package?: Neither, installed via Apt (version 1.0.0-7).
* System Python: 3.10.12

If you need more information, just let me know and I will provide it.

**Comment by jfabernathy** [comment]
I noticed that you are on Ubuntu 22.04.  I'm using Mint 21.2 which is based on 22.04 Jammy.  I'm using the ubuntu package for gourmand and it's working.  In fact I was on the flatpak version but I could not edit ingredients in some cases so I tried the system package.  It's working for now.

I see that the versions of Ubuntu newer than jammy are using the latest RC3 version of Gourmand.  

I hope someone is still around who can fix some of these issues.  There are just not any recipe programs on Linux and the ones of Windows cost $$$ and don't work most of the time.
